src/blocks/posts/inspector.js

```diff
@@ -74,6 +74,25 @@ const Inspector = props => {
 		},
 	];
 
+	const columnCountByPosts = ( selectedColumns ) => {
+		const { postsToShow } = attributes;
+		setAttributes( { columns:
+			( selectedColumns > postsToShow ) ? postsToShow : selectedColumns,
+		} );
+	};
+
+	const postCountByColumn = ( selectedPosts ) => {
+		const { columns } = attributes;
+		if ( columns > selectedPosts || ( selectedPosts === 1 && columns !== 1 ) ) {
+			setAttributes( {
+				columns: selectedPosts,
+				postsToShow: selectedPosts,
+			} );
+			return;
+		}
+		setAttributes( { postsToShow: selectedPosts } );
+	};
+
```

Comment:
See my comments in the [`post-carousel/inspector.js`](https://github.com/godaddy/coblocks/pull/1084/files#diff-2e5f26fcb8648e88a57c0ffad902109b) file as they apply here as well.